less/variables.less

```diff
@@ -13,14 +13,15 @@
 @footer-font-color: @waterloo;
 
 // Color Vars
-@wild-sand: #F6F6F6;
-@river-bend: #454A59;
-@sea-shell: #F1F1F1;
-@ebony-clay: #23293B;
-@shuttle-gray: #5E6164;
-@porcelain: #F6F7F8;
-@waterloo: #81848E;
-@btn-bg-color: #22283A;
+@wild-sand: #f6f6f6;
+@river-bend: #454a59;
+@sea-shell: #f1f1f1;
+@ebony-clay: #23293b;
+@shuttle-gray: #5e6164;
+@porcelain: #f6f7f8;
+@waterloo: #81848e;
+@btn-bg-color: #22283a;
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: (max-width: 500px);
```

Comment:
Create a @phone variable that contains a max-width: 500px media query string. Use the @phone variable for all your nested mobile styling. - DONE!

 Review the provided home mobile design file. Match your mobile styling the best you can using the design file. - WELL DONE, Dominic!